exec/RenderHtml.hs

```diff
@@ -378,7 +166,7 @@ refreshSamples = do
 runTidyIfAvailable :: FilePath -> IO ()
 runTidyIfAvailable fn' = withTidy >>= (`when` doTidy)
   where
-    fn'' = dropExtensions fn' <.> ".html-tidy.html"
+    fn'' = dropExtension fn' <.> ".tidy"
```

Comment:
I see that only my bash completion is fooled by .tidy, other programs, including `file` and `emacs` happily recognize this as a HTML file, despite it not ending in  .html.